backend/tournesol/views/preview.py

```diff
@@ -40,7 +43,9 @@
 COLOR_YELLOW_BACKGROUND = (255, 200, 0, 16)
 COLOR_WHITE_BACKGROUND = (255, 250, 230, 255)
 COLOR_BROWN_FONT = (29, 26, 20, 255)
+COLOR_WHITE_FONT = (255, 255, 255, 255)
 COLOR_NEGATIVE_SCORE = (128, 128, 128, 248)
+COLOR_DURATION_RECTANGLE = (0, 0, 0, 128)
```

Comment:
The background used on YouTube is `rgba(0, 0, 0, 0.8)`. I think it is the same as `(0,0,0,201)` ? Which should be darker

Reply:
ok! for my understanding, where do you get this value `0.8`? My reference was this https://github.com/tournesol-app/tournesol/blob/2512e03efe3b2dc3e2efc6191885235dadbdf536/frontend/src/components/entity/EntityImagery.tsx#L41 but I may not be in the right code.

Reply:
`0.8` is the value used on YouTube, but the design is quite different:

![image](https://user-images.githubusercontent.com/28259/215461061-c116f232-9b6e-4ede-926b-be28b2144d0d.png)

We should probably use the same style as the frontend, and maybe later update both to match YouTube's style.